src/Tribe/Cache_Listener.php

```diff
@@ -79,17 +85,43 @@ public function save_post( $post_id, $post ) {
 	     * @param mixed  $value     The new option value.
 		 */
 		public function update_last_save_post( $option_name, $old_value, $value ) {
-			$triggers = array(
-				'tribe_events_calendar_options',
-				'permalink_structure',
-				'rewrite_rules',
-				'start_of_week',
-			);
-			if ( in_array( $option_name, $triggers, true ) ) {
+			$triggers = [
+				'tribe_events_calendar_options' => true,
+				'permalink_structure'           => true,
+				'rewrite_rules'                 => true,
+				'start_of_week'                 => true,
+			];
+			if ( ! empty( $triggers[ $option_name ] ) ) {
```

Comment:
I'm not sure I understand why we changed the array here? There is no way to filter/change the triggers?
A simple array like before with an `isset` would seem to work just as well?

If this is to leave something in for easier testing somehow - maybe a comment to that effect would be useful?

Reply:
That array change is for optimization purposes. ([see this comment from Cris](https://github.com/moderntribe/tribe-common/pull/1253#discussion_r365923698))

It avoids us needing to do an `in_array()` call which is more costly than `isset()`.

Reply:
Gotcha - that makes sense